src/wp-admin/includes/ajax-actions.php

```diff
@@ -4036,8 +4036,9 @@ function wp_ajax_crop_image() {
 
 			/** This filter is documented in wp-admin/includes/class-custom-image-header.php */
 			$cropped    = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
-			$attachment = $wp_site_icon->create_attachment_object( $cropped, $attachment_id );
-			unset( $attachment['ID'] );
+
+			/* Copy attachment properties */
```

Comment:
```suggestion
			// Copy attachment properties.
```

For single line comment use these formate and add missing end period.